aquadoggo/src/graphql/client/dynamic_types/document_meta_type.rs

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+use async_graphql::indexmap::IndexMap;
+use async_graphql::{Name, OutputType, ScalarType, SelectionField, Value};
+use p2panda_rs::document::{DocumentId, DocumentViewId};
+
+use crate::graphql::client::dynamic_types::utils::{metafield, metaobject};
+use crate::graphql::scalars::{
+    DocumentId as DocumentIdScalar, DocumentViewId as DocumentViewIdScalar,
+};
+
+/// Name of the field for accessing the document's id.
+const DOCUMENT_ID_FIELD: &str = "documentId";
+
+/// Name of the field for accessing the document's view id.
+const DOCUMENT_VIEW_ID_FIELD: &str = "documentViewId";
```

Comment:
I like this being explicit but we had some ideas to change the API to `nextArgs(publicKey, viewId`), so it would be good to find some consistent naming for document view ids across the API. `viewId` is also nice, its so short?

Reply:
I like that!